web/share/update-languages.py

```diff
@@ -124,32 +118,31 @@ def update(self, po_files, lang2territory: str, threshold: int):
                 supported.append(locale)
 
         languages = [loc.language for loc in supported]
-        self.content["locales"] = dict()
         for locale in supported:
             include_territory = languages.count(locale.language) > 1
-            self.content["locales"][locale.code()] = locale.name(
-                include_territory)
+            self.content[locale.code()] = locale.name(include_territory)
 
     def write(self):
         with open(self.path, "w+") as file:
-            json.dump(self.content, file, indent=4, ensure_ascii=False)
+            json.dump(self.content, file, indent=4, ensure_ascii=False,
+                      sort_keys=True)
 
 
-def update_manifest(args):
+def update_languages(args):
     """Command to update the manifest.json file"""
-    manifest = Manifest(Path(args.manifest))
+    languages = Languages(Path(args.file))
     paths = [path for path in Path(args.po_directory).glob("*.po")]
     with open(args.territories) as file:
         lang2territory = json.load(file)
-    manifest.update(paths, lang2territory, args.threshold)
-    manifest.write()
+    languages.update(paths, lang2territory, args.threshold)
+    languages.write()
 
 
 if __name__ == "__main__":
-    parser = ArgumentParser(prog="locales.py")
-    parser.set_defaults(func=update_manifest)
+    parser = ArgumentParser(prog="update-languages.py")
+    parser.set_defaults(func=update_languages)
     parser.add_argument(
-        "manifest", type=str, help="Path to the manifest file",
+        "file", type=str, help="Path to the output file",
```

Comment:
I would call it output instead of file as it is more similar to input file.

Reply:
or maybe even better write to stdout by default and let user redirect it where he needs?